### ISSUE TITLE
RAFT small

### ISSUE THREAD

**Issue by antoniopavlis** [princeton-vl/RAFT#93]
Hi,
Do you maybe have some results (epe on chairs test or sintel) after training your --small model just on FlyingChairs ? Does additional training on FlyingThings has big impact ?

**Comment by hmorimitsu** [comment]
I don't know if it helps you, since I reimplemented part of the code myself (although it should still use the same procedure as this original RAFT). After training RAFT small on FlyingChairs only, and then using 12 iterations during inference, I observed the following EPE results in the training sets:

- Sintel Clean: 2.7
- Sintel Final: 3.9
- KITTI 2012: 5.3
- KITTI 2015: 11.8

**Comment by antoniopavlis** [comment]
Thank you
________________________________
Šalje: Henrique Morimitsu ***@***.***>
Poslano: 11. srpnja 2021. 9:52
Prima: princeton-vl/RAFT ***@***.***>
Kopija: Antonio Pavliš ***@***.***>; Author ***@***.***>
Predmet: Re: [princeton-vl/RAFT] RAFT small (#93)


I don't know if it helps you, since I reimplemented part of the code myself (although it should still use the same procedure as this original RAFT). After training RAFT small on FlyingChairs only, and then using 12 iterations during inference, I observed the following EPE results in the training sets:

  *   Sintel Clean: 2.7
  *   Sintel Final: 3.9
  *   KITTI 2012: 5.3
  *   KITTI 2015: 11.8

**Comment by plateau1** [comment]
> I don't know if it helps you, since I reimplemented part of the code myself (although it should still use the same procedure as this original RAFT). After training RAFT small on FlyingChairs only, and then using 12 iterations during inference, I observed the following EPE results in the training sets:
> 
> * Sintel Clean: 2.7
> * Sintel Final: 3.9
> * KITTI 2012: 5.3
> * KITTI 2015: 11.8

Could you provide the parameters you use to train the small model on the FlyingCharis dataset?Such as learning rate, Batchsize, imagesize and so on.Thank you very much.

**Comment by hmorimitsu** [comment]
I just used the same parameters used to train the larger model.
